public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
+  "avatar": "https://github.com/ShlokMohanty.png",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "http://github.com/ShlokMohanty",
```

Comment:
```suggestion
      "url": "https://github.com/ShlokMohanty",
```

Reply:
@Panquesito7 i guess you made the change from https to http or it still needs to be done ?

Reply:
No @ShlokMohanty, He just suggested.

Reply:
I also recommending that.

Reply:
@FayasNoushad  should i also resolve this conversation as well ?